tests/migrations/test_commands.py

```diff
@@ -695,6 +695,40 @@ def test_sqlmigrate_for_non_transactional_databases(self):
             self.assertNotIn(start_transaction_sql.lower(), queries)
         self.assertNotIn(connection.ops.end_transaction_sql().lower(), queries)
 
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_squashed"})
+    def test_sqlmigrate_ambigious_squashed_migration_name(self):
+        out = io.StringIO()
+        msg = "More than one migration matches '0001' in app 'migrations'. Please be more specific."
+        with self.assertRaisesMessage(CommandError, msg):
+            call_command("sqlmigrate", "migrations", "0001", stdout=out)
+
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_squashed"})
+    def test_sqlmigrate_replaced_migration(self):
+        out = io.StringIO()
+        call_command("sqlmigrate", "migrations", "0001_initial", stdout=out)
+        output = out.getvalue().lower()
+        self.assertIn('-- create model author', output)
+        self.assertIn('create table', output)
+        self.assertIn('-- create model tribble', output)
+
+    @override_settings(MIGRATION_MODULES={"migrations": "migrations.test_migrations_squashed"})
+    def test_sqlmigrate_replaced_second_migration(self):
+        out = io.StringIO()
+        call_command("sqlmigrate", "migrations", "0002_second", stdout=out)
+        output = out.getvalue().lower()
+        self.assertIn('-- delete model tribble', output)
+        self.assertIn('-- create model book', output)
```

Comment:
It looks that `test_sqlmigrate_replaced_second_migration()` and `test_sqlmigrate_replaced_migration()` are redundant. Please remove one of them.